projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx

```diff
@@ -74,14 +74,37 @@ export class DotDateRangeComponent {
         }
     ];
 
-    @Prop() presetLabel = 'Presets';
+    /** (optional) Text to be rendered next to presets field */
+    @Prop({ reflectToAttr: true }) presetLabel = 'Presets';
 
     @State() status: DotFieldStatus;
 
     @Event() valueChange: EventEmitter<DotFieldValueEvent>;
     @Event() statusChange: EventEmitter<DotFieldStatusEvent>;
 
     private fp: any;
+    private defaultPresets = [
+        {
+            label: 'Date Presets',
+            days: 0
+        },
+        {
+            label: 'Last Week',
+            days: -7
+        },
+        {
+            label: 'Next Week',
+            days: 7
+        },
+        {
+            label: 'Last Month',
+            days: -30
+        },
+        {
+            label: 'Next Month',
+            days: 30
+        }
+    ];
```

Comment:
I don't know if we should take this to another file

Reply:
IMO not needed since at this moment this will be only used by this component